### DISCUSSION TITLE
Branches Panel is Gone

### DISCUSSION THREAD

**Discussion by jbosse** [gitkraken/vscode-gitlens#3858] [Q&A] [answered]
It appears that the "Branches Panel" under "Source Control" is no longer there and apparently this was provided by GitLens. It would seem this is the only feature from GitLens I used/needed and now that it's missing I am very discombobulated. Is there just a setting I need to know to put it back? If not, is there a reason it was removed? If so, can I fork v15.6, remove everything but that and make my own extension?
![Screenshot 2024-12-11 at 8 17 31 AM](https://github.com/user-attachments/assets/af787731-a606-44ec-9db8-0c6b2e53e8d0)

**Comment by sergiolms** [comment, marked as answer]
Hey @jbosse ! Thanks for posting here. 

We've grouped all GitLens panels into a single one. You can find the branches tab under the GitLens panel (the one at the bottom of your screenshot).
![image](https://github.com/user-attachments/assets/445149cd-5610-46c5-bdbc-65e7f322d874)

You can also detach the panel and have it always visible as before if you prefer to.
![Captura de pantalla 2024-12-11 a las 16 07 12](https://github.com/user-attachments/assets/17287f89-529e-4d5c-8ace-3264dab30863)

Hope that helps

**Reply by jbosse** [reply]
Thank you. I would never have found that 😬 Leaving it on branches or even detaching is perfect for my needs. Crisis averted 😅
